Modules/_sre.c

```diff
@@ -511,6 +510,14 @@ state_getslice(SRE_STATE* state, Py_ssize_t index, PyObject* string, int empty)
     } else {
         i = STATE_OFFSET(state, state->mark[index]);
         j = STATE_OFFSET(state, state->mark[index+1]);
+
+        /* check wrong span */
+        if (i > j) {
+            PyErr_SetString(PyExc_RuntimeError,
+                            "the span of capturing group is wrong,"
+                            " please report a bug.");
```

Comment:
It is not idiomatic to raise a `RuntimeError` stating "please report a bug".
I think an assertion or `SystemError` should probably be used instead.

Reply:
I changed to `SystemError`.